tests/code_style.py

```diff
@@ -0,0 +1,88 @@
+"""RDT Contributing test module."""
+
+import importlib
+import inspect
+import json
+import re
+from pathlib import Path
+
+from rdt.transformers.base import BaseTransformer
+
+
+def validate_transformer_name(transformer):
+    """Return whether or not the ``Transformer`` ends with the ``Transformer`` nomenclature."""
+    return transformer.__name__.endswith('Transformer')
+
+
+def validate_transformer_subclass(transformer):
+    """Return whether or not the ``Transformer`` is a subclass of ``BaseTransformer``."""
+    return issubclass(transformer, BaseTransformer)
+
+
+def validate_transformer_module(transformer):
+    """Return whether or not the ``Transformer`` is inside the right module."""
+    transformer_file = Path(inspect.getfile(transformer))
+    transformer_folder = transformer_file.parent
+
+    is_valid = False
+    if transformer_folder.match('transformers'):
```

Comment:
By this logic, it seems like all these are valid:
- `rdt/transformers/my_module.py`
- `rdt/transformers/my_family/my_module.py`
- `rdt/transformers/addons/my_addon/my_module.py`

What about `rdt/transformers/addons/my_addon/my_family/my_module.py`?
Should it be valid too?

Reply:
I guess it depends how we want to organize things. I could see the possibility of having a broad `add-on` that has multiple sub-families in it, so I think this should be valid. For example, a location `add-on` that has a `family` for addresses and another one for geolocation or something like that.